Games/Retro_ML.SuperMarioKart/Neural/Scoring/TimeTakenScoreFactor.cs

```diff
@@ -11,6 +12,11 @@ internal class TimeTakenScoreFactor : IScoreFactor
         private double currScore;
         private int levelFrames = 0;
 
+        public FieldInfo[] Fields => new FieldInfo[]
+        {
+             new IntegerFieldInfo(nameof(MaximumRaceTime), "Maximum Race Time", 1, int.MaxValue, 1)
```

Comment:
Should be Double, but increment can remain at 1.0. Minimum Race Time should be 30.0